plugins/mcreator-localization/lang/texts.properties

```diff
@@ -784,7 +784,8 @@ blockly.block.compare_dimensionids=%1 = %2
 blockly.block.compare_directions=%1 = %2
 blockly.block.compare_entities=%1 = %2
 blockly.block.compare_mcblock_material=Is %2 %1 material type
-blockly.block.compare_mcblocks=%3 %1 = %4 %2
+blockly.block.compare_mcblocks=Is %3 %1 the same block as %4 %2
+blockly.block.compare_blockstates=Is %3 %1 the same blockstate as %4 %2
```

Comment:
One more note: to follow alphabetical order, this should be moved before `blockly.block.compare_dimensionids`.